specificatie/openapi.yaml

```diff
@@ -695,6 +699,19 @@ components:
           type: number
       explode: false
 
+    pandIdentificaties:
```

Comment:
Willen we pandIdentificaties als een comma-separated string wordt opgegeven (in de browser, postman)? Dan moet explode: false als attribuut worden toegevoegd, conform locatie-query